tests/base.py

```diff
@@ -66,7 +66,6 @@
 MODELS = (User, Node, NodeLog, NodeWikiPage,
           Tag, WatchConfig, Session, Guid)
 
-
```

Comment:
This line should not have been removed.

From [pep8](https://www.python.org/dev/peps/pep-0008/#blank-lines):

> Surround top-level function and class definitions with two blank lines.